### DISCUSSION TITLE
how we perform api call form relationship entity?

### DISCUSSION THREAD

**Discussion by dunods** [strapi/strapi#6389] [Legacy Topics]
for exapmle there are product-category and product, product-category has many products?

as strapi mechanism will produce

> localhost:1337/product-category/1
> localhost:1337/product/1

can strapi perform this
> localhost:1337/product-category/1/product
to fetch product under product category 

any advice on that
many thanks

**Comment by lauriejim** [comment]
Hello if you want, you can already do this `/product?category.id=1`
This will fetch all articles related to the category ID 1 (in this example category is the name of the relation)
Related to the [filter documentation](https://strapi.io/documentation/v3.x/content-api/parameters.html#deep-filtering)

After that, if you want your custom route, you will have to go in you `api` folder and create a custom route for your API.
Here is the [related documentation](https://strapi.io/documentation/v3.x/concepts/routing.html#example)
Then fetch the data by forcing the relation.
